### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jM5557**

### CODE REVIEW

File: UI/PauseScreen.cpp [hunk 1]
```diff
@@ -385,7 +385,10 @@ void GamePauseScreen::CreateViews() {
 		I18NCategory *mm = GetI18NCategory("MainMenu");
 		rightColumnItems->Add(new Choice(mm->T("Exit")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
 	} else {
-		rightColumnItems->Add(new Choice(pa->T("Exit to menu")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
+		I18NCategory *mm = GetI18NCategory("MainMenu");
```
Comment (unknownbrackets) [inline UI/PauseScreen.cpp]: This is a shared_ptr now, use `auto mm =`.

-[Unknown]

File: UI/PauseScreen.cpp [hunk 2]
```diff
@@ -449,11 +449,11 @@ UI::EventReturn GamePauseScreen::OnExit(UI::EventParams &e) {
...
-#ifdef __ANDROID__
-#ifdef ANDROID_NDK_PROFILER
-	moncleanup();
-#endif
-#endif
+	#ifdef __ANDROID__
+	#ifdef ANDROID_NDK_PROFILER
+		moncleanup();
+	#endif
+	#endif
```
Comment (unknownbrackets) [inline UI/PauseScreen.cpp]: We do not prefer indenting preprocessor directives.  That said, this block can probably be removed anyway.

-[Unknown]

File: UI/PauseScreen.h [hunk 3]
```diff
@@ -40,7 +40,11 @@ class GamePauseScreen : public UIDialogScreenWithGameBackground {
 private:
 	UI::EventReturn OnGameSettings(UI::EventParams &e);
 	UI::EventReturn OnExitToMenu(UI::EventParams &e);
+<<<<<<< HEAD
 	UI::EventReturn OnExit(UI::EventParams &e);
+=======
+	UI::EventReturn OnExit(UI::EventParams & e);
+>>>>>>> 42a94311d842f6e06589632bc449a98745ae565d
```
Comment (unknownbrackets) [inline UI/PauseScreen.h]: This is a merge conflict.

-[Unknown]

File: UI/PauseScreen.cpp [hunk 4]
```diff
@@ -385,7 +385,10 @@ void GamePauseScreen::CreateViews() {
 		I18NCategory *mm = GetI18NCategory("MainMenu");
 		rightColumnItems->Add(new Choice(mm->T("Exit")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
 	} else {
-		rightColumnItems->Add(new Choice(pa->T("Exit to menu")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
+		I18NCategory *mm = GetI18NCategory("MainMenu");
+
+		rightColumnItems->Add(new Choice(pa->T("Exit to Menu")))->OnClick.Handle(this, &GamePauseScreen::OnExitToMenu);
+		rightColumnItems->Add(new Choice(mm->T("Exit to Desktop")))->OnClick.Handle(this, &GamePauseScreen::OnExitToDesktop);
```
Comment (unknownbrackets) [inline UI/PauseScreen.cpp]: Definitely don't want this on all platforms, especially as "Desktop". Also, what's the rationale? Why are you adding this?

Also, you changed it to `OnExitToDesktop`, but it's still `OnExit` elsewhere.

-[Unknown]

Comment (jM5557) [reply UI/PauseScreen.cpp]: Thank you for the replies and feedback! This commit showed up on your repo by accident due to a complication with git bash. I'll be implementing the changes you suggested and proceeding with work on a separate repository that won't interfere with this one. Apologies for the inconvenience.

**Pull request closed without merging**